src/auth/jwt.strategy.ts

```diff
@@ -17,8 +17,9 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
   async validate(payload: JwtPayload): Promise<UserDto> {
     const user = await this.authService.validateUser(payload);
     if (!user) {
-      throw new UnauthorizedException();
+      throw new UnauthorizedException('Invalid token');
```

Comment:
Let's not use String directly. Add it to a file of constants containing error messages. Make a folder for containing constants.